__tests__/spelling-ignore.yml

```diff
@@ -59,6 +61,9 @@
 - html52
 - whatwg
 - dom
+- wai-aria-1.1
+- dfn-labels
```

Comment:
I am not a fan of having made up strings like this used for hyperlinks be added :(.

Reply:
But I think this should be solved in our testing engine for spelling rather than here.

Reply:
Yes, spelling should recognise URL and not parse them (this is a job for URL liveness checker (https://github.com/act-rules/act-rules-web/issues/21). Also, spelling should not need exception for each of the "G1", … "G128", … techniques and so on. Issue is already opened: #929